C2.md

```diff
@@ -63,11 +63,11 @@ be requested via C2 DESCRIBE manifest command.
 	# specify classes for the AST response
 	nifi.c2.root.classes=DeviceInfoNode,AgentInformation,FlowInformation
 	
-	# specify C2 protocol -- default is CoapProtocol if not specified
+	# specify C2 protocol -- default is RESTSender if not specified
+	nifi.c2.agent.protocol.class=RESTSender
 	#nifi.c2.agent.protocol.class=CoapProtocol
 	# may also use MQTT or REST
 	# nifi.c2.agent.protocol.class=MQTTC2Protocol
```

Comment:
Thanks for the PR, I think we should also change the non-default help comments here as well to reflect the change
```suggestion
	# may also use MQTT or CoapProtocol
	# nifi.c2.agent.protocol.class=MQTTC2Protocol
	# nifi.c2.agent.protocol.class=CoapProtocol
```